releasenotes/notes/fix_symbol_expr-28bc9af30c97388f.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:

```

Comment:
We don't actually need a release note for this. We haven't released 2.1.0 yet so there hasn't been a release with `SymbolExpr` yet.